ci-operator/step-registry/assisted/baremetal/operator/assisted-baremetal-operator-chain.yaml

```diff
@@ -3,7 +3,12 @@ chain:
   steps:
     - ref: assisted-baremetal-conf-devscripts-ipstack
     - ref: assisted-baremetal-conf-devscripts-extraworkers
-    - chain: baremetalds-ipi-pre
+    - ref: baremetalds-packet-setup
+    - ref: ipi-install-rbac
+    - ref: ipi-install-hosted-loki
+    - ref: assisted-baremetal-operator-configure-host
```

Comment:
@osherdp this is what I was telling you last time, I had to copy the `baremetalds-ipi-pre` chain in order to inject the `assisted-baremetal-operator-configure-host` step that will partition the disks. Do you see a better way to do that?

I thought about having the partition script as part of dev-scripts but I think it's too specific to our use case and it doesn't makes much sense to have it in there.

Reply:
Maybe have it in dev-scripts steps (or as a separate step included in dev-scripts chain) instead of dev-scripts repo?
Because I can see how it can benefit ipi jobs and have a good scoping in this place